app/src/main/java/de/grobox/transportr/favorites/trips/FavoriteTripPopupMenu.java

```diff
@@ -47,6 +57,9 @@ public boolean onMenuItemClick(MenuItem item) {
 				}
 				setFavState(item, trip.isFavorite());
 				return true;
+			case R.id.action_add_shortcut:
+				addShortcut(null, getShortcutName(), R.mipmap.ic_launcher_favorite);
```

Comment:
What is a `null` type? If `null` makes sense here, you need to annotate the parameter `@Nullable`.